src/processor.ts

```diff
@@ -123,13 +124,14 @@ export class D2Processor {
   };
 
   async generatePreview(source: string, signal: AbortSignal): Promise<string> {
+    const pathDelimiter = delimiter;
```

Comment:
Actually, why resassign to pathDelimiter? Why not just join on the import?

Reply:
@berniexie that may be a better way to do this. My development background isn't in Node/Web development so some of the more intimate details of the language/framework are not apparent to me. Want me to correct this, or you want to override yourself?

Reply:
@berniexie I have made the requested changed. Cheers! 🍻